### DISCUSSION TITLE
Example for Union Types

### DISCUSSION THREAD

**Discussion by Poolshark** [wp-graphql/wp-graphql#1772] [Q&A] [answered]
I just started to implement GraphQL Union Types into my project but I'm really confused of how to do this properly. One thing puzzles me especially: how do I get access to the Type Registry?

I'm not sure if it makes sense to add code snippets at this stage, but I could attach some if requested.

Thank's a lot! Any help appreciated!

**Comment by jasonbahl** [comment, marked as answer]
@Poolshark I would start with the documentation for `register_graphql_union` here: https://www.wpgraphql.com/functions/register_graphql_union_type/

There's some explanation of their utility and some code snippets that should help you get started adding Unions to your Schema.

**Reply by Poolshark** [reply]
@jasonbahl thanks for the link! I really have looked through the docs but somehow I must have missed this. Sorry!

Thanks for pointing me in the right direction!
